core-bundle/src/Cron/PurgePreviewLinksCron.php

```diff
@@ -35,7 +35,7 @@ public function __construct(Connection $connection)
     public function __invoke(): void
     {
         $this->connection->executeStatement(
-            'DELETE FROM tl_preview_link WHERE createdAt<=UNIX_TIMESTAMP(DATE_ADD(NOW(), INTERVAL 31 DAY))'
+            'DELETE FROM tl_preview_link WHERE createdAt<=UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL 31 DAY))'
```

Comment:
```suggestion
            'DELETE FROM tl_preview_link WHERE createdAt<=UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL 31 DAY)) AND expiresAt<=UNIX_TIMESTAMP(NOW())'
```
_(untested)_

Reply:
This change would make it easy to create a preview link that lives indefinitely, and I don't think we should ever allow for that for security reasons.

Reply:
I have created a separate ticket hereto (see #5184).